components/Layout/Navbar.tsx

```diff
@@ -35,39 +28,45 @@ export const Navbar = (): JSX.Element => {
           alt="Mutual Aid NYC"
         />
       </Link>
-      <HStack spacing="32px" pr={4} display={{ base: 'none', md: 'inherit' }}>
-        <Link href="/">Resource Lists</Link>
-        <Link href="https://resources.mutualaid.nyc/">Resource Directory</Link>
-        <Link href="/create-list">Create New List</Link>
+      <HStack
+        spacing={{ base: '16px', lg: '32px' }}
+        display={{ base: 'none', md: 'inherit' }}
+      >
+        <Link href="/">Resource lists</Link>
```

Comment:
nit: Design wise does the capitalization matter?

Reply:
Personally I prefer sentence case because that's what we use at work but [according to this article Apple prefers title case](https://uxdesign.cc/title-case-vs-sentence-case-in-ux-writing-212087192261) 😛 
We can switch it back to title case if people prefer that though.